### ISSUE TITLE
Tutorial - Adding a sticky header or banner

### ISSUE THREAD

**Issue by westnz** [WordPress/Learn#1435]
# Topic Description

Users can now position the Navigation Bar to stick to the top of the screen by nesting the Header inside a Group block and selecting “Sticky” from the Position option.

# Related Resources
Links to related content on Learn, HelpHub, DevHub, GitHub Gutenberg Issues, DevNotes, etc.
- [ ]

# Guidelines
Review the [team guidelines](https://make.wordpress.org/training/handbook/guidelines/)

# Tutorial Development Checklist

- [x] Vetted by instructional designers for content idea
- [x] Provide feedback of the idea
- [x] Gather links to Support and Developer Docs
- [x] Review any related material on Learn
- [x] Define several SEO keywords to use in the article and where they should be prominently used
- [x] Description and Objectives finalized
- [x] Tutorial [created](https://make.wordpress.org/training/handbook/tutorials/creating-a-tutorial) and announced to the team for Q/A review
- [x] Tutorial reviewed and [ready to publish](https://make.wordpress.org/training/handbook/tutorials/publishing-a-tutorial)
- [x] Tutorial [submitted and published to WPTV](https://make.wordpress.org/training/handbook/tutorials/publishing-a-tutorial/#1-submit-your-video-to-wordpress-tv) 
- [x] Tutorial published on WPTV
- [x] Tutorial [captioned](https://make.wordpress.org/training/handbook/tutorials/publishing-a-tutorial/#2-generate-subtitles-and-a-transcript-for-your-video) 
- [x] Tutorial [created on Learn.WordPress.org](https://make.wordpress.org/training/handbook/tutorials/publishing-a-tutorial/#3-create-edit-your-workshop-post)
- [x] Tutorial post reviewed for grammar, spelling, etc.
- [x] Tutorial published on Learn.WordPress.org
- [x] Tutorial [announced to Marketing Team](https://docs.google.com/spreadsheets/d/1rY9gyA0KljjlinPE2B0S2qfsN8Nz24_N_KzqB5CrYZM/edit#gid=469255212) for promotion

**Comment by courtneyr-dev** [comment]
Hey @westnz, the weird work around in this release is to add the header template part inside a group from the template page (Home, etc). Once in a group block, it'll be sticky.  Less ideal, but should see a fix in 6.3

**Comment by westnz** [comment]
Yes, that's correct, Courtney.  One thing I also noticed is changing the Group block's background colour; otherwise, it is transparent. Thanks!

**Comment by westnz** [comment]
https://user-images.githubusercontent.com/89751452/227386362-8dbe595e-e71f-47b9-9a06-34e486eba24c.mp4

**Comment by wparasae** [comment]
This looks fantastic, Wes! Concise and easy to follow. 

Optional addition: The only “gotcha” I’ve had with this feature is that sometimes the group block adds extra padding (Twenty Twenty-Three does this). You might consider  mentioning that it may take some adjustments to the group and navigation block to get the desired effect.

**Comment by westnz** [comment]
Recording: https://learn.wordpress.org/tutorial/adding-a-sticky-header-or-banner/

**Comment by courtneyr-dev** [comment]
WP 6.4
Patterns: add a sticky header and pagination to make browsing patterns more user friendly ([52663](https://github.com/WordPress/gutenberg/pull/52663#top)).

**Comment by courtneyr-dev** [comment]
WP 6.4 It is now also available for others patterns
[Sticky header bar on Patterns page](https://make.wordpress.org/core/2023/07/26/whats-new-in-gutenberg-16-3-26-july/#sticky-header-bar-on-patterns-page). Does it need a full revision @westnz ?

**Comment by westnz** [comment]
Hey @courtneyr-dev 
Thanks for flagging this again. I got confused as the link you shared is not about creating sticky headers, but only about the header bar on the Patterns page in the Site Editor being sticky.  

I see now that patterns will also be able to become sticky, so a full revision is required. 👍

**Comment by westnz** [comment]
This lesson is open for anyone to work on and update.

**Comment by westnz** [comment]
I will possibly update and reuse as part of the User learning pathway.

**Comment by Piyopiyo-Kitsune** [comment]
@westnz Do you think we will be reusing this?

**Comment by westnz** [comment]
Yes, as a standalone lesson but it needs to be updated at some point.
